prometheus_hardware_exporter/__main__.py

```diff
@@ -3,7 +3,11 @@
 import argparse
 import logging
 
-from .collector import MegaRAIDCollector
+from .collector import (
+    LSISAS2ControllerCollector,
```

Comment:
Not blocking. However it would be useful to consider using class decorators and store all collectors in an array.

Reply:
Thanks for suggestions. Can you explain more why using a class decorator it useful in this context? As for storing all collectors in an array, I think it might be better to leave it as it is right now since I would expect that in future there will be configuration option to enable / disable individual collector, so it might not be useful to combine them into a single array

Reply:
as an example
```python
registry = []

def register(decorated):
    registry.append(decorated)
    return decorated

@register
def foo():
    return 3

@register
def bar():
    return 5
```
 then in main

```python
for collector in registry
    exporter.register(collector)
```
if there is a config option in future you can just edit the register function to consider config values. In any case you will not need to modify another file when you add a new collector.

Reply:
Okay, that make senses. We should probably include this for future PR when config option for enable / disable collectors is added.